springwolf-examples/springwolf-amqp-example/src/test/resources/asyncapi.yaml

```diff
@@ -0,0 +1,264 @@
+        bindingVersion: 0.3.0
+  example-topic-routing-key:
+    messages:
+      io.github.springwolf.examples.amqp.dtos.AnotherPayloadDto:
+        $ref: '#/components/messages/io.github.springwolf.examples.amqp.dtos.AnotherPayloadDto'
+    bindings:
+      amqp:
+        is: routingKey
+        exchange:
+          name: example-bindings-exchange-name
+          type: topic
+          durable: true
+          autoDelete: false
+          vhost: /
+        bindingVersion: 0.3.0
+  multi-payload-queue:
+    messages:
+      io.github.springwolf.examples.amqp.dtos.AnotherPayloadDto:
+        $ref: '#/components/messages/io.github.springwolf.examples.amqp.dtos.AnotherPayloadDto'
+      io.github.springwolf.examples.amqp.dtos.ExamplePayloadDto:
+        $ref: '#/components/messages/io.github.springwolf.examples.amqp.dtos.ExamplePayloadDto'
+    bindings:
+      amqp:
+        is: routingKey
+        exchange:
+          name: example-topic-exchange
+          type: topic
+          durable: true
+          autoDelete: false
+          vhost: /
+        bindingVersion: 0.3.0
+components:
+  schemas:
+    HeadersNotDocumented:
+      type: object
+      properties: {}
+      description: "There can be headers, but they are not explicitly documented."
+      examples:
+        - {}
+    SpringRabbitListenerDefaultHeaders:
+      type: object
+      properties: {}
+      examples:
+        - {}
+    io.github.springwolf.examples.amqp.dtos.AnotherPayloadDto:
+      type: object
+      properties:
+        example:
+          $ref: '#/components/schemas/io.github.springwolf.examples.amqp.dtos.GenericPayloadDtoIo.github.springwolf.examples.amqp.dtos.ExamplePayloadDto'
+        foo:
+          type: string
+          description: Foo field
+          examples:
+            - bar
+      description: Another payload model
+      examples:
+        - example:
+            payload:
+              localDate: "2015-07-20"
+              localDateTime: 2015-07-20T15:49:04-07:00
+              someEnum: FOO2
+              someLong: 5
+              someString: some string value
+          foo: bar
+      required:
+        - example
+    io.github.springwolf.examples.amqp.dtos.ExamplePayloadDto:
+      type: object
+      properties:
+        localDate:
+          type: string
+          format: date
+        localDateTime:
+          type: string
+          format: date-time
+        someEnum:
+          type: string
+          description: Some enum field
+          enum:
+            - FOO1
+            - FOO2
+            - FOO3
+          examples:
+            - FOO2
+        someLong:
+          type: integer
+          description: Some long field
+          format: int64
+          examples:
+            - 5
+        someString:
+          type: string
+          description: Some string field
+          examples:
+            - some string value
+      description: Example payload model
+      examples:
+        - localDate: "2015-07-20"
```

Comment:
generated yaml does contain `localDate: 2015-07-20` (without double quotes) but this is reported as an error by studi.asyncapi.com
This does not necessarily means this is actually an error, given the fact that next line ( `localDateTime: 2015-07-20T15:49:04-07:00` )is  NOT reported as an error by studio.asyncapi.com ....